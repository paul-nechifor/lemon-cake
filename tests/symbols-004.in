(list (quote asdf) 1 (quote ab))
