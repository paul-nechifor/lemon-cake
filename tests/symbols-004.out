(asdf 1 ab)
